src/cli_parser.cpp

```diff
@@ -394,6 +410,11 @@ void CLIParser::prepareServer(ServerSettingsImpl& serverSettings) {
     if (result->count("trace_path"))
         serverSettings.tracePath = result->operator[]("trace_path").as<std::string>();
 #endif
+
+    serverSettings.allowCredentials = result->operator[]("allow_credentials").as<bool>();
```

Comment:
Should we validate that only if allowCredentials  is set then other options are allowed ? Or is it not dependent ?

Reply:
it is not dependent, we can enable/disable any of those